dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -199,6 +201,19 @@ public fun DataFrame.Companion.readSqlQuery(connection: Connection, sqlQuery: St
     }
 }
 
+/** SQL-query is valid only if it starts from SELECT and doesn't contain any other DDL or DML or DCL operator.*/
+private fun isValid(sqlQuery: String): Boolean {
+    val normalizedSqlQuery = sqlQuery.trim().uppercase()
+
+    return normalizedSqlQuery.startsWith("SELECT") &&
+        !normalizedSqlQuery.contains(";") &&
+        !forbiddenKeywords.any { normalizedSqlQuery.contains(it) }
+}
+
+private val forbiddenKeywords = setOf("INSERT", "UPDATE", "DELETE", "LOCK",
```

Comment:
sorry, not so forbidden:
```sql
CREATE TABLE "ALTER"
(
    id SERIAL PRIMARY KEY,
    description TEXT
);
```
is valid

Reply:
so
```sql
SELECT * from "ALTER";
```
is too

Reply:
Could you explain your comment? Do you mean that given example are passed the condition? Or should be accepted as an SQL queries?

Reply:
`SELECT * from "ALTER"` is a valid SQL query and should be accepted, however, since you check whether the query doesn't contain any of these keywords, it won't be accepted atm.